src/clim2parquet/__init__.py

```diff
@@ -0,0 +1,103 @@
+# SPDX-FileCopyrightText: 2025-present Imperial College of Science and Technology
+#
+# SPDX-License-Identifier: MIT
+
+from . import tools
+
+import os
+from typing import Union
+import warnings
+
+
+def get_data_names():
+    """
+    Get data source names.
+
+    Returns
+    -------
+    list
+        A list of data source names.
+    """
+    return tools._get_data_info()["data_source"].tolist()
+
+
+def clim_to_parquet(
+    data_source: Union[str, list],
```

Comment:
Usually this is 

```suggestion
    data_source: str | list[str],
```

so both the type of what the list contains and `|` rather than `Union`.  This might impose a minimum python restriction of ~3.10 I think, but that's not a huge issue with the python release schedule: https://devguide.python.org/versions/ and it's what we've gone for to simplify typing in other projects